### PR TITLE
Added more documentation

### PR DESCRIPTION
**Pull request by joeribekker**
Assumes #88 is merged.

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -13,10 +13,12 @@ black==20.8b1             # via -r requirements/dev.in
 certifi==2020.4.5.1       # via -r requirements/ci.txt, elastic-apm, requests, sentry-sdk
 chardet==3.0.4            # via -r requirements/ci.txt, requests
 click==7.1.2              # via black, pip-tools
+commonmark==0.9.1         # via recommonmark
 coreapi==2.3.3            # via -r requirements/ci.txt, drf-yasg
 coreschema==0.0.4         # via -r requirements/ci.txt, coreapi, drf-yasg
 coverage==4.5.4           # via -r requirements/ci.txt
 cssselect==1.1.0          # via -r requirements/ci.txt, pyquery
+dataclasses==0.7          # via black
```
Comment (annashamray) [inline requirements/dev.txt:21]: Do you use python 3.7 or above? They already include `dataclasses` in standard lib for python >= 3.7, therefore `pip install -r requirements/dev.txt` fails with error now

File: docs/introduction/definitions.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+An API to retrieve one or more *objecttypes*.
+
+The Objecttypes API aims to standardize various types of objects, on a national
+or just as easily on a local municipality level, and make them accessible as
+resources for other applications.
+
+Objects API
+-----------
+
+An API to retrieve, filter, search, write or update *objects*.
```
Comment (annashamray) [inline docs/introduction/definitions.rst:36]:
```suggestion
An API to retrieve, filter, search, write, update or delete *objects*.
```

File: docs/introduction/visualization.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
+Visualization
+=============
+
+The Objectypes API works together with the Objects API, objects of a certain
```
Comment (annashamray) [inline docs/introduction/visualization.rst:4]: It looks like the second part of this sentence is not finished?

File: docs/introduction/visualization.rst [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+Ofcourse, not only the municipality of Utrecht can do this. Many more 
+organizations can do this and store the trees that are relevant for them 
+according to this definition.
+
+.. image:: _assets/multiple_objects_apis_using_national_objecttypes_api.png
+    :width: 100%
+    :alt: More organizations store their trees according to the national definition.
+
+Every organization is free to decide which objecttypes to use but also, which 
+Objecttypes API to use. It's fully supported to run your own Objecttypes API '
```
Comment (annashamray) [inline docs/introduction/visualization.rst:30]:
```suggestion
Objecttypes API to use. It's fully supported to run your own Objecttypes API 
```

File: docs/introduction/features.rst [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+---------------
+
+* **JSON-schema validation**
+
+  When creating a new objecttype, the JSON-schema is validated. Only valid 
+  JSON-schemas are allowed in the objecttype.
+
+* **Versioning**
+
+  Objecttypes are versioned. This means you can create an objecttype but it the
```
Comment (annashamray) [inline docs/introduction/features.rst:16]:
```suggestion
  Objecttypes are versioned. This means you can create an objecttype but if the
```

File: docs/installation/config.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
+# Environment configuration reference
+
+Open Personen can be ran both as a Docker container or directly on a VPS or
```
Comment (annashamray) [inline docs/installation/config.md:3]:
```suggestion
The Objects and Objecttypes API can be ran both as a Docker container or directly on a VPS or
```

**Review by annashamray — COMMENTED** [review]
Documentation looks great, especially the visualization part!

File: requirements/dev.txt (hunk 1 above)
Comment (joeribekker) [reply requirements/dev.txt:21]: Woops, I used Python 3.6

File: docs/introduction/visualization.rst (hunk 3 above)
Comment (joeribekker) [reply docs/introduction/visualization.rst]: Slight change made.

**Review by annashamray — APPROVED** [review]

**Pull request merged**